scripts.js

```diff
@@ -5,28 +5,60 @@ const close1 = document.getElementById('close');
 const close2 = document.getElementById('close2');
 const close3 = document.getElementById('close3');
 
-btn.onclick = function () {
+btn.onclick = function openButt() {
   modal.style.display = 'block';
 };
 
-span.onclick = function () {
+span.onclick = function spanClose() {
   modal.style.display = 'none';
 };
 
-close1.onclick = function () {
+close1.onclick = function button1Close() {
   modal.style.display = 'none';
 };
 
-close2.onclick = function () {
+close2.onclick = function button2Close() {
   modal.style.display = 'none';
 };
 
-close3.onclick = function () {
+close3.onclick = function button3Close() {
   modal.style.display = 'none';
 };
 
-window.onclick = function (event) {
+window.onclick = function outerClose(event) {
   if (event.target === modal) {
     modal.style.display = 'none';
   }
+};
+
+function upperCaseErrorMessage() {
+  const messageContainer = document.getElementById('errorMessage');
+  messageContainer.innerHTML = '<b>Error</b>: Email should only contain lower cases.';
+}
+
+function checkEmailUppercase() {
+  let email = document.getElementById('email').value;
+  email = email.split('');
+
+  for (let i = 0; i < email.length; i += 1) {
+    if (email[i].match(/[a-z]/i)) {
+      if (email[i] === email[i].toUpperCase()) {
+        const errorContainer = document.getElementById('errorMessage');
+        errorContainer.style.display = 'block';
+        upperCaseErrorMessage();
+
+        return false;
+      }
+    }
+  }
+  return true;
```

Comment:
- [ ] Kindly I would suggest arranging the error message in the mobile version. Check here down :point_down: 
![Screenshot from 2021-12-17 00-02-03](https://user-images.githubusercontent.com/65068771/146456404-dd0723cd-8dea-4895-a774-37a93efba45b.png)
- [ ] I would recommend also to put the error message in red color.

Reply:
I have fixed them c: